src/components/MerchantsPage/MerchantsPage.tsx

```diff
@@ -130,3 +137,18 @@ const MerchantInfoContainer = styled.div`
     flex-direction: column-reverse;
   }
 `;
+
+const FlyerContainer = styled.div`
+  margin: 3vw 7vw;
+  font-weight: bold;
+  
+  a {
```

Comment:
I personally prefer to define classes instead of styling tags, but seems like we have no standard here

Reply:
Either way works for me, I can add a class to be more specific

Reply:
In the case that we make all of our links look like this, then it can be the a tag. But if it's just isolated to this one section, then I'd say use a class. Now thinking of it, we should probably  use the tag and pull it out into a shared style